src/mmrelay/matrix_utils.py

```
@@ -2216,19 +2259,19 @@ def _get_e2ee_error_message():
-    meshtastic_id=None,
-    meshtastic_replyId=None,
-    meshtastic_text=None,
-    emote=False,
-    emoji=False,
-    reply_to_event_id=None,
-):
+    room_id: str,
+    message: str,
+    longname: str,
+    shortname: str,
+    meshnet_name: str,
```

Ruff (0.14.10)

Unused function argument: `meshnet_name`

(ARG001)